core/src/test/java/io/grpc/internal/ManagedChannelImplIdlenessTest.java

```diff
@@ -205,15 +204,14 @@ public void enterIdleModeAfterForceExit() throws Exception {
     newTransports.drainTo(transports);
     assertEquals(addressGroupList.size(), transports.size());
 
-    InterimTransport<ClientTransport> interimTransport = channel.tm.createInterimTransport();
+    channel.tm.createInterimTransport();
```

Comment:
That is your own personal preference and it does not need to apply to these tests. I can see the value, but I don't think it is a universal rule, since many tests have to do lots of stuff I would never want a real user to do. Preventing a single one-line change that is consistent with the class because the class was not written as you would like does not seem acceptable.